Linking problem using Alire

I tried to port [jfuica/bingada](https://github.com/jfuica/bingada) to use Alire. The default sound option is canberra_ada, which I saw it's already in the index.

So I made the following to create a new project file for Alire from scratch:

```shell
mv bingada.gpr bingada_custom.gpr
alr init --in-place --bin bingada
# new bingada.gpr edited to adapt compiler switches for bingada
alr with canberra_ada
alr with gtkada
alr build
```
I expected it to build without any further configuration, but I got a linking problem.
```
Link
   [link]         bingada.adb
/usr/bin/ld: /home/mgr/src/github/jfuica/bingada/alire/cache/dependencies/canberra_ada_2.0.0_3de867fe/build/lib/libcanberra-ada.a(canberra.o): in function `canberra__set_property':
canberra.adb:(.text.canberra__set_property+0x6f): undefined reference to `ca_context_change_props'
/usr/bin/ld: /home/mgr/src/github/jfuica/bingada/alire/cache/dependencies/canberra_ada_2.0.0_3de867fe/build/lib/libcanberra-ada.a(canberra.o): in function `canberra__set_property__2':
canberra.adb:(.text.canberra__set_property__2+0x5f): undefined reference to `ca_proplist_sets'
/usr/bin/ld: /home/mgr/src/github/jfuica/bingada/alire/cache/dependencies/canberra_ada_2.0.0_3de867fe/build/lib/libcanberra-ada.a(canberra.o): in function `canberra__finalize__2':
canberra.adb:(.text.canberra__finalize__2+0xe): undefined reference to `ca_context_destroy'
/usr/bin/ld: /home/mgr/src/github/jfuica/bingada/alire/cache/dependencies/canberra_ada_2.0.0_3de867fe/build/lib/libcanberra-ada.a(canberra.o): in function `canberra__create':
canberra.adb:(.text.canberra__create+0x133): undefined reference to `ca_context_create'
/usr/bin/ld: /home/mgr/src/github/jfuica/bingada/alire/cache/dependencies/canberra_ada_2.0.0_3de867fe/build/lib/libcanberra-ada.a(canberra.o): in function `canberra__play_internal':
canberra.adb:(.text.canberra__play_internal+0x3e): undefined reference to `ca_proplist_create'
/usr/bin/ld: canberra.adb:(.text.canberra__play_internal+0x17d): undefined reference to `ca_context_play_full'
/usr/bin/ld: canberra.adb:(.text.canberra__play_internal+0x1f3): undefined reference to `ca_proplist_destroy'
/usr/bin/ld: canberra.adb:(.text.canberra__play_internal+0x21d): undefined reference to `ca_proplist_destroy'
/usr/bin/ld: /home/mgr/src/github/jfuica/bingada/alire/cache/dependencies/canberra_ada_2.0.0_3de867fe/build/lib/libcanberra-ada.a(canberra.o): in function `canberra__cancelCL':
canberra.adb:(.text.canberra__cancelCL+0xc): undefined reference to `ca_context_cancel'
collect2: error: ld returned 1 exit status
gprbuild: link of bingada.adb failed
gprbuild: failed command was: /usr/bin/gnatgcc bingada.o b__bingada.o /home/mgr/src/github/jfuica/bingada/obj/q_bingo.o /home/mgr/src/github/jfuica/bingada/obj/q_bingo-q_gtk.o /home/mgr/src/github/jfuica/bingada/obj/q_bingo-q_gtk-q_intl.o /home/mgr/src/github/jfuica/bingada/obj/q_bingo_help.o /home/mgr/src/github/jfuica/bingada/obj/q_csv.o /home/mgr/src/github/jfuica/bingada/obj/q_csv-q_read_file.o /home/mgr/src/github/jfuica/bingada/obj/q_gen_shuffle.o /home/mgr/src/github/jfuica/bingada/obj/q_sound.o /home/mgr/src/github/jfuica/bingada/obj/q_bingo-q_bombo.o /home/mgr/src/github/jfuica/bingada/obj/q_bingada.o /home/mgr/src/github/jfuica/bingada/alire/cache/dependencies/gtkada_21.0.0_3c1373c0/install_dir/lib/gtkada/gtkada.static/gtkada/libgtkada.a /home/mgr/src/github/jfuica/bingada/alire/cache/dependencies/canberra_ada_2.0.0_3de867fe/build/lib/libcanberra-ada.a -Linstall_dir/lib -Linstall_dir/bin -lgtk-3 -lgdk-3 -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -Wl,--export-dynamic -lgmodule-2.0 -pthread -lglib-2.0 -lfontconfig -lfreetype -L/home/mgr/src/github/jfuica/bingada/obj/ -L/home/mgr/src/github/jfuica/bingada/obj/ -L/home/mgr/src/github/jfuica/bingada/alire/cache/dependencies/canberra_ada_2.0.0_3de867fe/build/lib/ -L/home/mgr/src/github/jfuica/bingada/alire/cache/dependencies/gtkada_21.0.0_3c1373c0/install_dir/lib/gtkada/gtkada.static/gtkada/ -L/usr/lib/gcc/x86_64-linux-gnu/9/adalib/ -shared-libgcc -lgnarl-9 -lgnat-9 -lpthread -lrt -ldl -Wl,-rpath-link,/usr/lib/gcc/x86_64-linux-gnu/9//adalib -Wl,-z,origin,-rpath,$ORIGIN/..//obj/install_dir/lib:$ORIGIN/..//obj/install_dir/bin:$ORIGIN/..//obj:$ORIGIN/..//alire/cache/dependencies/canberra_ada_2.0.0_3de867fe/build/lib:$ORIGIN/..//alire/cache/dependencies/gtkada_21.0.0_3c1373c0/install_dir/lib/gtkada/gtkada.static/gtkada:/usr/lib/gcc/x86_64-linux-gnu/9/adalib:/usr/lib64:/usr/lib -o /home/mgr/src/github/jfuica/bingada/bin//bingada
error: Command ["gprbuild", "-gnatwU", "-j0", "-p", "-P", "bingada.gpr"] exited with code 4
error: Compilation failed.
```

Is some `-lcanberra` missing somewhere? For gtkada, a switch like `-lgtk-3` is added to the link command, but nothing about `-lcanberra`

--- a/canberra_ada.gpr
+++ b/canberra_ada.gpr
@@ -95,4 +95,8 @@ project Canberra_Ada is
       for Switches ("Ada") use ("-Es"); --  Symbolic traceback
    end Binder;
 
+   package Linker is
+      for Linker_Options use ("-lcanberra");
+   end Linker;
+
 end Canberra_Ada;

